### PR TITLE
FFTF-226 : Privacy statement 

### PR DESCRIPTION
**Pull request by diya2214**

### CODE REVIEW

File: app/assets/src/layout.njk [hunk 1]
```diff
@@ -13,46 +16,56 @@
 
 {% block header %}
   {{ govukHeader({
-    homepageUrl: "/",
-    serviceName: "Check if you can apply for a water resource management grant",
-    serviceUrl: "/"
+  homepageUrl: "/",
+  serviceName: "Check if you can apply for a water resource management grant",
```
Comment (govTechSatish) [inline app/assets/src/layout.njk:20]: water resource management grant => Farming Transformation Fund water grant

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-frontend",
-  "version": "1.46.0",
+  "version": "1.47.0",
```
Comment (govTechSatish) [inline package.json:3]: change version to 1.46.1

**Review by govTechSatish — CHANGES_REQUESTED** [review]
Please check comments

File: package.json (hunk 2 above)
Comment (diya2214) [reply package.json:3]: its a new feature we use last digit after second decimal for bug fixes only

**Pull request closed without merging**